### PR TITLE
added security group to output

### PR DESCRIPTION
**Pull request by Suhas009**

### CODE REVIEW

File: emr.tf [hunk 1]
```diff
@@ -920,3 +942,6 @@ resource "aws_route53_record" "hbase_replica" {
   records  = [aws_emr_cluster.hbase_read_replica.master_public_dns]
 }
 
+output "replica_emr_hbase_common" {
+  value = aws_security_group.replica_emr_hbase_common
+}
```
Comment (TheDogfather17) [inline emr.tf]:
```suggestion
}

```

File: emr.tf [hunk 2]
```diff
@@ -661,6 +661,28 @@ resource "aws_security_group_rule" "egress_to_vpce" {
...
+//  //todo: move to internal-compute vpc module
+//  description              = "Allow mysql traffic to Aurora RDS from ADG"
+//  from_port                = 3306
+//  protocol                 = "tcp"
+//  security_group_id        = data.terraform_remote_state.adg.outputs.adg_common_sg.id
+//  to_port                  = 3306
+//  type                     = "egress"
+//  source_security_group_id = aws_security_group.replica_emr_hbase_common.id
+//}
+
```
Comment (TheDogfather17) [inline emr.tf]:
```suggestion
```

**Review by TheDogfather17 — CHANGES_REQUESTED** [review]

File: emr.tf (hunk 1 above)
Comment (Suhas009) [reply emr.tf]: Implemented suggestion.

File: emr.tf (hunk 2 above)
Comment (Suhas009) [reply emr.tf]: Implemented suggestion.

**Review by TheDogfather17 — APPROVED** [review]

**Pull request merged**